API.md

```diff
@@ -120,3 +120,16 @@ Calls `sendBytes()` with `{ 0x00 }` as payload to poll for incoming messages.
 Returns the result of `sendBytes()`.
 
 See the [Receive](https://github.com/TheThingsNetwork/arduino-device-lib/blob/master/examples/Receive/Receive.ino) example.
+
+## Method: provision
+Sets the information needed to activate the device.
```

Comment:
".. the device via OTAA, without actually activating. Call `join()` without the first 2 arguments to activate."